### ISSUE TITLE
Add option to set font size in view mode

### ISSUE THREAD

**Issue by umm7yass** [gsantner/markor#609]
I have the font size increased in editor mode for readability, but then once I go back to the representation mode the font is small again and harder to read.

**Comment by gsantner** [comment]
Yoh have full control over that. How everything looks in representation.

Use representation injection option and set e.g.:

* { font-size: 150%; }

**Comment by umm7yass** [comment]
@gsantner Are you talking about "Inject-> head" and "Inject-> body" because I've tried both. I tried the following to zero effect whatsoever:

1. edited the existing one in the head option to 400%
1. pasting the following into into the body option: full contents of the head option with the percent modified
1. pasting the item you suggested into the body option verbatim

I'm not sure what I'm doing wrong.
